projects/news_feed/utilities.py

```diff
@@ -17,29 +17,111 @@
 # global variables to use
 PSL_URL_REQUEST = "https://www.espncricinfo.com/scores/series/8679/season/2020/pakistan-super-league?view=results"
 
+# requesting to access PSL website
+url_response = requests.get(PSL_URL_REQUEST)
+source_code = url_response.content
 
-def grab_psl_matches_urls(class_attribute, soup_layout_format):
+# creating Beautiful siup object
+soup = BeautifulSoup(source_code, 'lxml')
```

Comment:
why these lines are moved here?

Reply:
These lines were inside function that stores match links. Now I have removed this function.

Reply:
ok then rename them with capital letters e.g. SRC_SOUP, SRC_CODE and URL_RESPONSE

its general python rule to name global variables defined at top of file of outside functions to be in snake casing and capital letters.

to know more about snake casing and other naming conventions in coding check [this ](https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841)

Reply:
@M-Shan Global variables renamed in snake case (All caps)